src/main/java/ch/njol/skript/util/PotionEffectUtils.java

```diff
@@ -341,4 +334,74 @@ public static List<PotionEffect> getEffects(ItemType itemType) {
 		return effects;
 	}
 
+	public static ClassInfo<PotionEffectType> getLegacyClassInfo() {
+		return new ClassInfo<>(PotionEffectType.class, "potioneffecttype")
+			.user("potion( ?effect)? ?types?") // "type" had to be made non-optional to prevent clashing with potion effects
+			.name("Potion Effect Type")
+			.description("A potion effect type, e.g. 'strength' or 'swiftness'.")
+			.usage(StringUtils.join(getNames(), ", "))
+			.examples("apply swiftness 5 to the player",
+				"apply potion of speed 2 to the player for 60 seconds",
+				"remove invisibility from the victim")
+			.since("")
+			.supplier(PotionEffectType.values())
+			.parser(new Parser<>() {
+				@Override
+				@Nullable
+				public PotionEffectType parse(final String string, final ParseContext context) {
+					return parseType(string);
+				}
+
+				@Override
+				public String toString(final PotionEffectType potionEffectType, final int flags) {
+					return PotionEffectUtils.toString(potionEffectType, flags);
+				}
+
+				@Override
+				public String toVariableNameString(final PotionEffectType potionEffectType) {
+					return toString(potionEffectType, 0);
+				}
+			})
+			.serializer(new Serializer<>() {
+				@Override
+				public Fields serialize(final PotionEffectType potionEffectType) {
+					final Fields f = new Fields();
+					f.putObject("name", potionEffectType.getKey().getKey());
+					return f;
+				}
+
+				@Override
+				public boolean canBeInstantiated() {
+					return false;
+				}
+
+				@Override
+				public void deserialize(final PotionEffectType o, final Fields f) {
+					assert false;
+				}
```

Comment:
```suggestion
```
Can be removed while we're here

Reply:
When I remove it, it tells me I need it.

Reply:
Maybe it's using a different serializer class. Not a huge deal though, we can ignore this.